outputs/exec.go

```diff
@@ -0,0 +1,55 @@
+package outputs
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+
+	"github.com/aquasecurity/postee/v2/layout"
+)
+
+type execCmd = func(string, ...string) *exec.Cmd
+
+type ExecClient struct {
+	ExecCmd   execCmd
+	Name      string
+	Env       []string
+	InputFile string
+	Output    []byte
+}
+
+func (e *ExecClient) GetName() string {
+	return e.Name
+}
+
+func (e *ExecClient) Init() error {
+	e.ExecCmd = exec.Command
+	e.Name = "Exec Output"
+	return nil
+}
+
+func (e *ExecClient) Send(m map[string]string) error {
+	envVars := os.Environ()
+	envVars = append(e.Env, fmt.Sprintf("POSTEE_EVENT=%s", m["description"]))
```

Comment:
it looks like value of `envVars`  assigned on line 33 is overwritten

Reply:
thanks great catch - I fixed it.